sickbeard/providers/newznab.py

```diff
@@ -125,7 +125,6 @@ def get_providers_list(data):
                 providers_dict[default.name].enable_daily = default.enable_daily
                 providers_dict[default.name].enable_backlog = default.enable_backlog
                 providers_dict[default.name].enable_manualsearch = default.enable_manualsearch
-                providers_dict[default.name].catIDs = default.catIDs
```

Comment:
This was the cause for an issue where the category id's are always overwritten, after a restart. You could change the categories in the provider_config. But a restart would revert it back to 5030,5040